features/tests/e2e/setup/electron.js

```diff
@@ -119,6 +119,19 @@ After({ tags: '@restartApp' }, async function() {
   context.app = await startApp();
 });
 
+// this ensures that the reset-backend call successfully executes
+// after the app version difference test sets the app to disconnected state
+// eslint-disable-next-line prefer-arrow-callback
+After({ tags: '@reconnectApp' }, async function() {
```

Comment:
@daniloprates @tomothespian @MarcusHurney @DominikGuzei I have added this new tag to our e2e setup as new test for "Manual software update overlay" pushes Daedalus into disconnected state which causes all the tests which are ran after it to fail due to a fact that `Backend-Reset` Before step call fails (it expects Daedalus to be connected).

Reply:
✅